--- docs/configure.vanilla	2004-07-04 22:48:44.122701632 +0200
+++ docs/configure	2004-07-04 22:51:38.653168968 +0200
@@ -70,12 +70,12 @@
 #VIRTUAL_DOMAINS = SELECT DISTINCT domain || ' : ' FROM domains WHERE type = 'local'
 #RELAY_DOMAINS = SELECT DISTINCT domain || ' : ' FROM domains WHERE type = 'relay'
 
-domainlist local_domains = @ : pear.silverwraith.com : ${lookup mysql{VIRTUAL_DOMAINS}} : ${lookup mysql{ALIAS_DOMAINS}}
+domainlist local_domains = @ : pldmachine.pld-network.net : ${lookup mysql{VIRTUAL_DOMAINS}} : ${lookup mysql{ALIAS_DOMAINS}}
 domainlist relay_to_domains = ${lookup mysql{RELAY_DOMAINS}}
 hostlist   relay_from_hosts = localhost : 192.168.0.0/24
-trusted_users = avleen:www
+trusted_users = exim:http
 
-hide mysql_servers = localhost::(/tmp/mysql.sock)/vexim/vexim/CHANGE
+hide mysql_servers = localhost::(/var/lib/mysql/mysqldb/mysql.sock)/vexim/vexim/CHANGE
 #hide pgsql_servers = localhost/vexim/vexim/CHANGE
 
 # Most straightforward access control requirements can be obtained by
@@ -145,7 +145,7 @@
 # virus scanning, leave it commented. Please read doc/exiscan-acl-readme.txt
 # for a list of supported scanners.
 
-av_scanner = clamd:/var/run/clamav/clamd
+av_scanner = clamd:/var/lib/clamav/clamd.socket
 
 # The following setting is only needed if you use the 'spam' ACL condition
 # of the exiscan-acl patch. It specifies on which host and port the SpamAssassin
@@ -271,12 +271,12 @@
 ######################################################################
 
 begin acl
-  .include /usr/local/etc/exim/vexim-acl-check-spf.conf
+  .include /etc/mail/vexim-acl-check-spf.conf
 
 acl_check_helo:
 
 # Include Vexim specific helo ACLs
-  .include /usr/local/etc/exim/vexim-acl-check-helo.conf
+  .include /etc/mail/vexim-acl-check-helo.conf
 
 # This access control list is used for every RCPT command in an incoming
 # SMTP message. The tests are run in order until the address is either
@@ -290,7 +290,7 @@
   accept  hosts = :
 
   # Include Vexim specific rcpt ACLs
-  .include /usr/local/etc/exim/vexim-acl-check-rcpt.conf
+  .include /etc/mail/vexim-acl-check-rcpt.conf
 
   # Deny if the local part contains @ or % or / or | or !. These are rarely
   # found in genuine local parts, but are often tried by people looking to
@@ -374,7 +374,7 @@
 acl_check_content:
 
   # Include Vexim specific rcpt ACLs
-  .include /usr/local/etc/exim/vexim-acl-check-content.conf
+  .include /etc/mail/vexim-acl-check-content.conf
 
   # finally accept all the rest
   accept
